great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -542,6 +542,7 @@ def _check_linting(expectation_instance) -> ExpectationDiagnosticCheckMessage:
                 )
             else:
                 black_ok = True
+            # TODO: update this for ruff (or don't)
             isort_ok = isort.check_code(
```

Comment:
If we want to replace `isort` we need to replace this `expectation_diagnostics.py` check as well.

Reply:
It was decided in our architecture review meeting that we should just remove this whole method and rely on our normal linting/formatting tools to enforce this.